omrmakefiles/rules.linux.mk

```diff
@@ -195,6 +195,27 @@ ifeq (x86,$(OMR_HOST_ARCH))
         GLOBAL_CPPFLAGS+=-DJ9X86
     endif
 
+else ifeq (riscv,$(OMR_HOST_ARCH))
+        GLOBAL_CFLAGS  +=-DRISCV
+        GLOBAL_CXXFLAGS+=-DRISCV
+        GLOBAL_CPPFLAGS+=-DRISCV
+        ifeq (1,$(OMR_ENV_DATA64))
+             GLOBAL_CFLAGS  +=-DRISCV64
+             GLOBAL_CXXFLAGS+=-DRISCV64
+             GLOBAL_CPPFLAGS+=-DRISCV64
+        else
+             GLOBAL_CFLAGS  +=-DRISCV32
+             GLOBAL_CXXFLAGS+=-DRISCV32
+             GLOBAL_CPPFLAGS+=-DRISCV32
+        endif
+        GCC_MAJOR_VERSION := $(shell ($(CC) -dumpversion 2>/dev/null || echo 1) | cut -d. -f1)
+        # stringop-truncation and stringop-overflow are enabled since GCC 9.0
+        ifeq (,$(findstring $(GCC_MAJOR_VERSION),1 2 3 4 5 6 7 8))
+            GLOBAL_CFLAGS   +=-Wno-error=stringop-truncation -Wno-error=stringop-overflow
+            GLOBAL_CXXFLAGS +=-Wno-error=stringop-truncation -Wno-error=stringop-overflow
+            GLOBAL_CPPFLAGS +=-Wno-error=stringop-truncation -Wno-error=stringop-overflow
+       endif
```

Comment:
This should not be necessary now that #4603 and eclipse/openj9#7879 are merged.

Reply:
Removed.